### DISCUSSION TITLE
Helt With setup

### DISCUSSION THREAD

**Discussion by addee99** [dyc3/steamguard-cli#212] [Q&A]
Hey! I need some help With the setup. After downloading the files, how do I start it? I have Windows.
And i dont understand the thing With ”rust toolchain”. 

I have no coding experience :/ 

apprecite the help

**Comment by dyc3** [comment]
After you've downloaded `steamguard.exe` from the releases, place your `maFiles` folder inside `%APPDATA%\Roaming\steamguard-cli\`. The full path to your `manifest.json` should look like this: `%APPDATA%\Roaming\steamguard-cli\maFiles\manifest.json`

You need to run the app from the command line. There is no gui right now.

To do this:

1. Put `steamguard.exe` on your desktop for easy access.
2. Open Powershell or Command Prompt
3. Use the `cd` command to navigate to the directory where `steamguard.exe` is located
```
cd Desktop
```
4. You can then run it
```
.\steamguard.exe --help
```

**Comment by addee99** [comment]
I have downloaded this one: https://github.com/dyc3/steamguard-cli
I cant find the steamguard.exe on the release here https://github.com/dyc3/steamguard-cli/blob/master/docs/releases "404 - page not found".

**Comment by addee99** [comment]
I got the file now. But cant find this %APPDATA%\Roaming\steamguard-cli\

**Reply by dyc3** [reply]
In your file browser window, up at the top where it shows the file path, click it and put in in `%APPDATA%\Roaming`.

If you can't find the `steamguard-cli` folder, you'll obviously have to create it.

**Comment by addee99** [comment]
https://gyazo.com/fc24d50d83e1db329db0d0d9b781b32d

Does this look right? I created a folder called "steamguard-cli" then another folder inside that called "maFiles". Then I added the mafiles and manifest.json file.

**Reply by dyc3** [reply]
Yup! That looks right to me. You should be able to run steamguard-cli like we discussed before and it should work

**Reply by addee99** [reply]
Alright, will give it a try. Just one more question. Is it auto confirming every X second? Cause I just want to confirm whenever I use the app, and not confirm all the trades all the time. If you understand what I mean

**Reply by dyc3** [reply]
No, it doesn't auto confirm
